### PR TITLE
refactor: use bind api

### PR DESCRIPTION
**Pull request by cqh963852**
relate #3

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (dai-shi) [inline .vscode/launch.json:1]: Do we need to commit this file?

**Review by dai-shi — COMMENTED** [review]
Is this just combining the two apis, or adding some new capabilities?

File: .vscode/launch.json (hunk 1 above)
Comment (cqh963852) [reply .vscode/launch.json:1]: > Is this just combining the two apis, or adding some new capabilities?

just combining two apis.no new capabilities.



> Do we need to commit this file?

This file is vscode + jest debug config. This adds the ability to single-step debugging. I think it's developer friendly.

Comment (dai-shi) [reply .vscode/launch.json:1]: Hmm, but I can't review the config file as I don't use it...

File: __tests__/01_basic_spec.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import * as Y from 'yjs';
 import { proxy } from 'valtio/vanilla';
-import { bindProxyAndYMap, bindProxyAndYArray } from '../src/index';
+import { bindProxy } from '../src/index';
```
Comment (dai-shi) [inline __tests__/01_basic_spec.ts:3]: Can it be `bind`?
```suggestion
import { bind } from '../src/index';
```

File: jest.config.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import type { Config } from '@jest/types';
+
+const config: Config.InitialOptions = {
+  preset: 'ts-jest',
+  moduleNameMapper: {},
+};
+
+export default config;
```
Comment (dai-shi) [inline jest.config.ts:8]: as this is not a big file, I prefer to have it in package.json.

**Comment by dai-shi** [comment]
Other than that, it looks good. Although I didn't follow all the code, if CI tests pass, it should be okay.

**Review by dai-shi — APPROVED** [review]
Thanks for your contribution. Let's ship it.

**Pull request merged**